server/command.go

```diff
@@ -107,8 +122,10 @@ func (p *Plugin) runStartCommand(args *model.CommandArgs, user *model.User) (str
 		return "Error checking previous messages", nil
 	}
 
+	_, _, topic := p.parseCommand(args.Command)
```

Comment:
It seems a bit off how there are two calls the `parseCommand`, ignoring certain return values in each call. What are your thoughts on having `runStartCommand` accept a `topic` string parameter to avoid this function call here?

Reply:
Yeah that's a good point. Just pushed a change.